src/libraries/System.IO.FileSystem/tests/Base/FileGetSetAttributes.cs

```diff
@@ -91,7 +91,7 @@ private void AssertSettingInvalidAttributes(string path, FileAttributes attribut
             Assert.Equal(FileAttributes.Normal, GetAttributes(path));
         }
 
-        [Theory,
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNetCore)),
```

Comment:
Rather than use a conditional theory, can we just make it a regular theory and use SkipOnTargetPlatform to skip it on .NET Framework?  That's generally what we do elsewhere.

Reply:
Oops I missed this one. I changed all the others, as originally suggested in the other PR.